### DISCUSSION TITLE
Refiners loading performance issues

### DISCUSSION THREAD

**Discussion by premramus** [microsoft-search/pnp-modern-search#2839] [Q&A]
Problem Statement:
We have configured search through PnP search third party and search result webpart takes more than 23 seconds to load more than 3 lacs records . 
There is another issue in PnP search filter which is not smoothly functional due to large amount of records . 
Please let me know team member who have worked on PnP search performance and provided same solution for large amount of records.

**Comment by kasperbolarsen** [comment]
Please update this issue with the basic info from the issues form.
